StPetersburg/InEtn210320.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0"
+     xml:lang="en"
+     xml:id="InEtn210320"
+     type="mss">
+  <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Prayer to Jesus Christ, Doctrine of Mysteries</title>
```

Comment:
I believe that we once decided to give as title the transcribed titles of the main work, whenever possible (so for təmhərta ḫəbuʾāt only here), but there is a lot of general inconsistency regarding this in all our records, and I do not find this in the guidelines now. I will open an issue.

Reply:
done